src/java.base/share/classes/sun/security/ssl/SSLSessionContextImpl.java

```diff
@@ -170,6 +178,52 @@ public int getSessionCacheSize() {
         return cacheLimit;
     }
 
+    // Package-private, used only from SSLContextImpl::engineInit() to initialie currentKeyID.
+    void initCurrentKeyID(int keyID) {
+        currentKeyID = keyID;
+    }
+
+    // Package-private, used only from SSLContextImpl::getKey() to create a new session key.
+    int getCurrentKeyID() {
+        return currentKeyID;
+    }
+
+    private void cleanupSessionKeys() {
```

Comment:
What do you think if using CleanupStatelessKeys as the method name?

Reply:
Fine for me.